enterprise/api/v1/serializers.py

```diff
@@ -97,3 +97,14 @@ class Meta:
     user = UserSerializer()
     enterprise_customer = EnterpriseCustomerSerializer()
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True)
+
+
+class EnterpriseCourseEnrollmentSerializer(serializers.ModelSerializer):
+    """
+    Serializer for EnterpriseCourseEnrollment model.
+    """
+    class Meta:
+        model = models.EnterpriseCourseEnrollment
+        fields = (
+            'id', 'enterprise_customer_user', 'course_id', 'consent_granted'
```

Comment:
One nit here. Rather than accessing the `consent_granted` field directly from the DB, I'd like to have it access the `consent_available` method. This would provide more useful contextual information that the consumer will need. In addition, I'd like the `consent_needed` property to be available as a read-only field so that the consumer doesn't need to carry the business logic with it - that way, we can update the requirement in the edx-enterprise repo whenever we need to.